aces/Loris
# [Instrument] Delete Instrument Data

## Brief summary of changes

- Removed the two unlabeled checkboxes
- Added a modal window for confirmation of data deletion
- Changed the condition in the NDB_BVL_InstrumentStatus_ControlPanel.class.inc file

#### Testing instructions 

Test the confirm button and the cancel button. Make sure the data is deleted (you need to reload the page)

#### Link(s) to related issue(s)

#6218 

Thank you @maltheism  and @haoweiqiu  for the help.
